### ISSUE TITLE
dart pub get vnum fails

### ISSUE THREAD

**Issue by adsharma** [AmirKamali/Flutter_Vnum#7]
https://github.com/adsharma/py2many/pull/112/checks?check_run_id=2427684955

> Because every version of vnum depends on flutter any from sdk which is forbidden, vnum is forbidden.
> So, because tests depends on vnum any, version solving failed.

py2many is a transpiler. So we're sensitive to our CI runtimes and would like to minimize dependencies.
Specifically, would like to stick to the core language libraries instead of frameworks.

Thank you for building vnum! I couldn't get some of the python features to work with dart enums.

Needed for https://github.com/adsharma/py2many/issues/26

**Comment by AmirKamali** [comment]
I agree, this package depends on Flutter SDK rather than Dart SDK. I remember I investigated this before and didn’t have success. 
Your current changes have your library codes inside. I’m hoping to keep this library clean and independent of others. I’d be happy to merge your pull request if you can make the changes.

**Comment by adsharma** [comment]
Thank you for creating vnum. It's helpful for me to map python enums to dart enums. I don't have any change requests at this point. Request is to make it easier/faster to install for CI purposes (we support 7 languages across 3 platforms).

I'll ping this issue if I come across any improvements.
